docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
 
-FROM adoptopenjdk/openjdk11:x86_64-ubuntu-jre-11.0.11_9
```

Comment:
We will most likely update the image to jre 17. Does changing to `x86_64-ubuntu-jre-11.0.12_7` solves the issue for you? I'll try to reproduce it locally as well.